configmap/parse.go

```diff
@@ -144,6 +146,45 @@ func AsQuantity(key string, target **resource.Quantity) ParseFunc {
 	}
 }
 
+func AsNamespacedName(key string, target *types.NamespacedName) ParseFunc {
+	return func(data map[string]string) error {
+		var (
+			name      string
+			namespace string
+			raw       string
+			ok        bool
+		)
+
+		if raw, ok = data[key]; !ok {
+			return nil
+		}
+
+		v := strings.Split(raw, string(types.Separator))
+		switch len(v) {
+		// No '/' separator.
+		case 1:
+			name = v[0]
+		case 2:
+			name = v[1]
+			namespace = v[0]
+		default:
+			return fmt.Errorf("failed to parse %q: %s ", key, "expected 'namespace/name' format")
```

Comment:
seems like an odd way to build the error message, also trailing space.

Reply:
man - mondays